commit 83a52a082b4a85ded434184bbec89b4dae657953 (refs/changes/01/4601/3)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-08-10T15:54:27+01:00 (1 year, 2 months ago)
    
    MANTA-3795 nginx build broken on recent platform images
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/deps/nginx b/deps/nginx
index de2e377..f054a7d 160000
--- a/deps/nginx
+++ b/deps/nginx
@@ -1 +1 @@
-Subproject commit de2e377384c2b4d61eed75bbd289d07558ab0bbc
+Subproject commit f054a7df29b99932cbe50c1216d545029b2e769c
